styleguide/source/_patterns/01-molecules/06-components/12-hero-banner/_hero-banner.scss

```diff
@@ -54,7 +60,10 @@
 .hero-banner_title {
   background-color: rgba($white, 0.45);
   padding: 20px 20px 8px;
-  @include type($font-sans-serif, 28px, $font-weight-thin, 1.1428571429);
```

Comment:
Removing this results in title using serif below $bp-small breakpoint because h1 is set to use serif by default. 

Recommend adding @include type($font-sans-serif, 24px, $font-weight-thin, 1.2916666667); instead of just the font-size: 24px;